CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2020-02-06)
+
+- Fixed the phone field component to prevent autofocusing on phone field input when added to the page. [#32](https://github.com/blackbaud/skyux-phone-field/pull/32)
```

Comment:
The "when" clause seems weird to me. It makes it sound to me like the issue had to do with timing, and I don't think that's the case. Can we just delete that clause? Also, what do you think of reworking this a bit to:
`Fixed the phone field component so that focus is not automatically applied to the phone field input.`

Reply:
It was a timing thing actually... The bug only happened when the phone field was being shown the 2nd time. For example, if it was being controlled by an `ngIf`, and it was shown once, hidden, then shown again. The bug only occurred after the 2nd time it was shown. Any ideas on how best to word that?

Reply:
That's a tough one. Would this be too much detail?
`Fixed the phone field component so that focus is not automatically applied to the phone field input when it is added back to a page after previously being hidden.`

Reply:
Works for me. I've updated it.